src/modules/mga/mga.service.ts

```
@@ -100,97 +146,44 @@ export class MgaService {
-                const tooLongFields = stringFields.filter((field) => {
-                    const value = String(row[field]).trim();
-                    return value.length > 255; // Asumiendo límite de 255 caracteres
-                });
-
-                if (tooLongFields.length > 0) {
-                    errors.push(
-                        `Fila ${rowNumber}: Los siguientes campos exceden la longitud máxima (255 caracteres): ${tooLongFields.join(', ')}`,
-                    );
-                    return null;
+        const validatedData = data.map((row, index) => {
+            const rowNumber = index + 2; // +2 porque índice 0 es fila 1, pero hay encabezados
```

ESLint

[error] Delete `······`

(prettier/prettier)